Tests/vincentyTests/vincentyTests.swift

```diff
@@ -48,48 +61,78 @@ final class VincentyTests: XCTestCase {
 
         x = (lat: 0.asRad, lon: 0.asRad)
         y = (lat: 0.asRad, lon: 0.asRad)
-        XCTAssertEqual(try! vincenty.distance(x, y), 0.0, accuracy: delta)
+        XCTAssertEqual(try! vincenty.vincentyCalculations(x, y).distance, 0.0, accuracy: delta)
 
         x = (lat: 0.asRad, lon: 0.asRad)
         y = (lat: 1.asRad, lon: 0.asRad)
-        XCTAssertEqual(try! vincenty.distance(x, y), 110574.389, accuracy: delta)
+        XCTAssertEqual(try! vincenty.vincentyCalculations(x, y).distance, 110574.389, accuracy: delta)
 
         x = (lat: 0.asRad, lon: 0.asRad)
         y = (lat: 2.asRad, lon: 0.asRad)
-        XCTAssertEqual(try! vincenty.distance(x, y), 221149.453, accuracy: delta)
+        XCTAssertEqual(try! vincenty.vincentyCalculations(x, y).distance, 221149.453, accuracy: delta)
 
         x = (lat: 0.5.asRad, lon: 0.asRad)
         y = (lat: -0.5.asRad, lon: 0.asRad)
-        XCTAssertEqual(try! vincenty.distance(x, y), 110574.304, accuracy: delta)
+        XCTAssertEqual(try! vincenty.vincentyCalculations(x, y).distance, 110574.304, accuracy: delta)
 
         x = (lat: -0.5.asRad, lon: 0.asRad)
         y = (lat: 0.5.asRad, lon: 0.asRad)
-        XCTAssertEqual(try! vincenty.distance(x, y), 110574.304, accuracy: delta)
+        XCTAssertEqual(try! vincenty.vincentyCalculations(x, y).distance, 110574.304, accuracy: delta)
 
         x = (lat: 0.asRad, lon: 0.asRad)
         y = (lat: 0.asRad, lon: 1.asRad)
-        XCTAssertEqual(try! vincenty.distance(x, y), 111319.491, accuracy: delta)
+        XCTAssertEqual(try! vincenty.vincentyCalculations(x, y).distance, 111319.491, accuracy: delta)
 
         x = (lat: 0.asRad, lon: 0.asRad)
         y = (lat: 0.asRad, lon: 2.asRad)
-        XCTAssertEqual(try! vincenty.distance(x, y), 222638.982, accuracy: delta)
+        XCTAssertEqual(try! vincenty.vincentyCalculations(x, y).distance, 222638.982, accuracy: delta)
 
         x = (lat: 0.asRad, lon: 0.5.asRad)
         y = (lat: 0.asRad, lon: -0.5.asRad)
-        XCTAssertEqual(try! vincenty.distance(x, y), 111319.491, accuracy: delta)
+        XCTAssertEqual(try! vincenty.vincentyCalculations(x, y).distance, 111319.491, accuracy: delta)
 
         x = (lat: 0.asRad, lon: -0.5.asRad)
         y = (lat: 0.asRad, lon: 0.5.asRad)
-        XCTAssertEqual(try! vincenty.distance(x, y), 111319.491, accuracy: delta)
+        XCTAssertEqual(try! vincenty.vincentyCalculations(x, y).distance, 111319.491, accuracy: delta)
+        
+        //Test Cardinals
+        x = (lat: 0.0, lon: 0.0)
+        y = (lat: pi/2,lon: 0.0) //north pole
+        XCTAssertEqual(try! vincenty.vincentyCalculations(x, y).initialTrueTrack, 0.0, accuracy: delta)
```

Comment:
I guess adding some assertions (or test cases) on the `finalTrueTrack` would be helpful as well. So far, there are only tests on the `initialTrueTrack`.

Reply:
Yes, if only I had data to test with...

Reply:
I would try to keep the package domain unspecific. Some other domains may prefer to use [Gradian](https://en.wikipedia.org/wiki/Gradian) or some other units. If you look at trigonometric functions in the standard math library, they all work in rad. Also, I would try to not mix unit conversion with algorithms, following the Unix Philosophy "Make each program do one thing well".

Converting between rad and degrees or other units is a relatively trivial task, compared to solving Vincenty
's problem. So I think this is not something this package should handle. For testing, having convenience converters is fine, though.

Reply:
Tests on final track added on cardinal directions.
Sticking to radians.